module_3/gifts/build.gradle

```diff
@@ -0,0 +1,54 @@
+plugins {
+	id "org.springframework.boot" version "2.6.1"
+	id "io.spring.dependency-management" version "1.0.11.RELEASE"
+	id "war"
+}
+
+ext {
+	lombokVersion = '1.18.22'
+	jupiterVersion = '5.8.2'
+	mockitoVersion = '4.1.0'
+	jacksonDataVersion = '2.13.0'
+	springBootVersion = '2.6.1'
+}
+
+springBoot {
+    mainClass = 'com.epam.esm.Application'
+}
+
+allprojects {
+	group = 'com.epam.esm'
+	version = ''
+	sourceCompatibility = '15'
+
+	apply plugin: 'java-library'
+	
+	repositories {
+		mavenCentral()
+	}
+}
+
+subprojects {
+	dependencies {
+		compileOnly group: 'org.projectlombok', name: 'lombok', version: "${lombokVersion}"
+		annotationProcessor group: 'org.projectlombok', name: 'lombok', version: "${lombokVersion}"
+		
+		implementation group: 'org.springframework.boot', name: 'spring-boot-starter-parent', version: "${springBootVersion}"
+		implementation group: 'org.springframework.boot', name: 'spring-boot-starter', version: "${springBootVersion}"
+		implementation group: 'org.springframework.boot', name: 'spring-boot-starter-jdbc', version: "${springBootVersion}"	
+		implementation group: 'org.springframework.boot', name: 'spring-boot-starter-hateoas', version: "${springBootVersion}"
```

Comment:
better to move to web module

Reply:
@maximBortnik, I will move this dependency to the controller layer after removing RepresentationModel extension in Dto classes on service layer.